Composer/yarn.lock

```diff
@@ -2858,6 +2858,30 @@
     semver "^5.5.1"
     tslib "^1.10.0"
 
+"@microsoft/bf-lu@4.9.0-RC4":
```

Comment:
Why hasn't the previous entry been removed? Is it still being referenced somewhere?

Reply:
I have updated all related to RC4 now